### PR TITLE
Expeditions rework

### PR DESCRIPTION
**Pull request by metalgearsloth**
Really needs https://github.com/space-wizards/space-station-14/pull/18958

- Reduced timer to 6 minutes.
- Bumped mob count so it's harder to get through.
- Megafauna / structures are just side objective loot pinatas.
- Flat 15 minute cooldown and no hard requirement on finishing.
- Adding spawn markers for random stuff (so we don't need to rely on maps constantly being updated and they just have base ambient loot).
~~- Adding ambient mobs around if they decide to mine instead.~~
- Made it 1 difficulty so we can balance that first and reduce / increase accordingly once it's nailed down better.

Expectation is the whole dungeon won't be finishable but the loot density should be higher and fultons let you get items out without backtracking so every room has something worthwhile.

Old mob values:
image

New mob values (subject to final tweaks):
image

Tentative difficulty modifiers (working at tweaking a middle-ish difficulty atm):
image

UI:
image

:cl:
- tweak: Expeditions have been massively tweaked. Timer is 7 minutes, loot count is significantly up, added significantly more loot to the loot table, mob count is significantly up, and mobs overall have been nerfed (no more 15 damage carps ending you). It also has a flat 13 minute cooldown. Better bring your fultons if you expect to bring stuff back.

### CODE REVIEW

**Comment by mirrorcult** [comment]
ok we gotta stop calling everything a refactor

**Comment by metalgearsloth** [comment]
> ok we gotta stop calling everything a refactor

Alt-expeditions?

**Comment by mirrorcult** [comment]
epicxpeditions

**Comment by ThunderBear2006** [comment]
> ok we gotta stop calling everything a refactor

rework :)

**Comment by lzk228** [comment]
bro, lavalend already exist as idea :skull:

**Comment by KaiShibaa** [comment]
> bro, lavalend already exist as idea 💀

Trying to not BE like lavaland is the goal

File: Resources/Prototypes/Procedural/salvage_mods.yml [hunk 1]
```diff
@@ -65,195 +51,184 @@
...
+    - Lava
+
+- type: salvageAirMod
+  id: Breathable
+  cost: 0
+  gases:
+    - 21.824779 # oxygen
+    - 82.10312 # nitrogen
+
+# TODO: These are good but mobs don't entirely handle it yet.
```
Comment (deltanedas) [inline Resources/Prototypes/Procedural/salvage_mods.yml:134]: only atmos i see mobs having issues with is high temperatures, they dont breathe plasma or nitrous oxide so they dont get affected

**Comment by Xeno4** [comment]
how about adding syndicate faction to expeds? that would be a challenging thing

File: Resources/Prototypes/Procedural/salvage_mods.yml (hunk 1 above)
Comment (metalgearsloth) [reply Resources/Prototypes/Procedural/salvage_mods.yml:134]: sure.........

**Pull request merged**